src/unix/notbsd/linux/musl/mod.rs

```diff
@@ -0,0 +1,131 @@
+pub type clock_t = ::c_long;
+
+pub type blkcnt_t = i64;
+pub type fsblkcnt_t = u64;
+pub type fsfilcnt_t = u64;
+pub type ino_t = u64;
+pub type off_t = i64;
+pub type stat64 = ::stat;
```

Comment:
Hm ok, in the interest of more verification though could this struct be a redefinition using the `foo64_t` types? That way ctest will auto-check the members of the struct, size, alignment, etc. Right now it doesn't handle type aliases to structs like it does struct declarations themselves.